hack/verify-test-code.sh

```diff
@@ -42,6 +42,15 @@ do
     fi
 done
 
+errors_expect_equal=()
+for file in "${all_e2e_files[@]}"
+do
+    if grep "Expect(.*)\.To(.*Equal(" "${file}" > /dev/null
```

Comment:
The line detects test/e2e/auth/node_authn.go as containing errors.
node_authn.go contains the following line. But I think that it should not be detected.
```
gomega.Expect(result).To(gomega.Or(gomega.Equal("401"), gomega.Equal("403")), ...
```
How about changing it like `grep -E "Expect\(.*\)\.To\((gomega\.Equal|Equal)` ?

Reply:
@k-toyoda-pi Thank you for your comment, nice catch.
I will update this soon.